adserver/tests/test_decision_engine.py

```diff
@@ -613,9 +614,34 @@ def test_default_keywords(self):
             request=self.request, placements=self.placements, publisher=self.publisher
         )
 
-        self.assertTrue(
-            self.backend.keywords, ["foo", "bar", "baz", "machine-learning"]
+        self.assertEqual(
```

Comment:
While running the tests, I realized that this was just plain wrong. It was working but the test wasn't correctly checking it.